mobile/cmp/input/Select.scss

```diff
@@ -81,3 +81,37 @@
     }
   }
 }
+
+//---------------
+// Fullscreen
+//---------------
+#xh-select-input-fullscreen-portal {
+  z-index: 22;
+
+  .xh-select {
+    margin: var(--xh-pad-px);
+
+    &__fullscreen-wrapper {
+      position: absolute !important;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background: var(--xh-bg);
+    }
+
+    &__fullscreen-toolbar {
+      background-color: var(--xh-appbar-bg);
+      color: var(--xh-appbar-title-color);
+      padding: var(--xh-pad-half-px) var(--xh-pad-px) var(--xh-pad-half-px) 0;
```

Comment:
Seeing a funny artifact when testing on my iPhone 12, where the back button is stretched against the left edge of the toolbar.  Weird, as I see we have left padding specified...

![IMG_0615](https://user-images.githubusercontent.com/225806/123807205-a692f700-d8bd-11eb-9139-b67fa492ff59.PNG)

Reply:
Actually, that button is supposed to be transparent (similar to the main app bar colors)... will update the styling. Thanks for catching!